admindb/sqlite/invites.go

```diff
@@ -0,0 +1,207 @@
+package sqlite
+
+import (
+	"context"
+	"crypto/rand"
+	"crypto/sha256"
+	"database/sql"
+	"encoding/base64"
+	"fmt"
+
+	"github.com/friendsofgo/errors"
+	"github.com/mattn/go-sqlite3"
+	"github.com/volatiletech/null/v8"
+	"github.com/volatiletech/sqlboiler/v4/boil"
+	"github.com/volatiletech/sqlboiler/v4/queries/qm"
+
+	"github.com/ssb-ngi-pointer/go-ssb-room/admindb"
+	"github.com/ssb-ngi-pointer/go-ssb-room/admindb/sqlite/models"
+	refs "go.mindeco.de/ssb-refs"
+)
+
+// make sure to implement interfaces correctly
```

Comment:
could this scary comment be clarified within a parens? 
>  // make sure to implement interfaces correctly (e.g. don't use sqlboiler generated types in args/return values)

Reply:
Ah. These two lines are a compiler guarantee thing.  It just checks that the struct fulfills the interface. (You assign to an unused variable with a nil thing, which doesn't even allocate anything.)

meaning if you change the methods slightly this will fail to compile and tell you _I expected method X to return Y, not Z_.

Still open for suggestions how to phrase that better.

Reply:
ah ok yeah then maybe persisting what you wrote above as a comment could make sense? e.g.
```suggestion
// Checks that the struct fulfills the interface. (You assign to an unused variable with a nil thing, which doesn't even allocate anything.)
```

cause i was just spooked as hell by the comment, especially when my lack of internal project context was combined with the top-most package comment re sqlboiler generated variables